scripts/entrypoint.sh

```diff
@@ -15,4 +16,10 @@ if [ -n "${LAUNCHPAD_KEYCLOAK_URL}" ]; then
     sed -i.bckp 's#realm:.*,clientId#realm:"'${LAUNCHPAD_KEYCLOAK_REALM}'",url:"'${LAUNCHPAD_KEYCLOAK_URL}'",clientId#' ${APP_JS}
 fi
 
+if [ -n "${LAUNCHPAD_TRACKER_SEGMENT_TOKEN}" ]; then
+    sed -i.bckp 's#SegmentTrackerToken#${LAUNCHPAD_TRACKER_SEGMENT_TOKEN}#' ${INDEX}
+else
+    sed -i.bckp '/^.*<!-- Add segment/,/^.*<\/script/d' ${INDEX}
```

Comment:
I'm afraid this will not work as webpack will remove comments and white space from the build dist version. You should try the script against the version in the dist folder after running `npm run build:prod`